.github/workflows/release.yml

```diff
@@ -119,18 +119,17 @@ jobs:
           key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ runner.os }}-m2
 
-      - name: Maven release
-        id: vars
-        shell: bash
-        run: |
-          mvn -B -ntp \
-            -Drepository.url=https://${{ github.actor }}:${{ secrets.GITHUB_TOKEN }}@github.com/${{ github.repository }}.git \
-            release:prepare
-          mvn -pl :parent process-resources
-
       - name: Download all build artifacts
         uses: actions/download-artifact@v2
 
+      - name: Version
```

Comment:
If I take it right, this is adding a new commit to master with the new version being released. Wasn't this needed before?

Reply:
This is what the release:prepare goal did, among other things. I simply reduced it to the minimum.